### ISSUE TITLE
React.createElement: Type is invalid

### ISSUE THREAD

**Issue by ericuldall** [DennisSnijder/payload-workflow#7]
```
chunk-LAFHZV3R.js?v=839fdc55:136 Warning: React.createElement: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.
```

Using viteBundler

Is anyone else getting this error?

**Comment by DennisSnijder** [comment]
@ericuldall hi! thanks for reporing, I never tried the ViteBundler in combination with payload-workflow. I'll take a look if i can get it to work!

**Comment by jackadair** [comment]
@DennisSnijder any update on this? Still getting the same error on webpack.

**Comment by DennisSnijder** [comment]
> @DennisSnijder any update on this? Still getting the same error on webpack.

No update on this yet! Odd that you have this issue on webpack. Do you have some details about the setup you are using?

package manager, payload version etc? So i can try to recreate it :)

**Comment by DennisSnijder** [comment]
@ericuldall @jackadair this issue has been fixed in 0.4.0 🥳 
Thank you for reporting the issue!
